### PR TITLE
Publish dev builds to play store

### PR DESCRIPTION
**Pull request by m-kuhn**
A successful run from a different branch:

https://github.com/opengisch/QField/runs/1491226409?check_suite_focus=true

### CODE REVIEW

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for c1366de5d43ee261b0037970e5dfc7e1b62acb90:
  - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1445-94506714b2c0647d9eacffe9e85120c4aa7d72c4-arm64_v8a.apk)

Other architectures: [armv7](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1445-94506714b2c0647d9eacffe9e85120c4aa7d72c4-armv7.apk), [x86_64](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1445-94506714b2c0647d9eacffe9e85120c4aa7d72c4-x86_64.apk), [x86](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1445-94506714b2c0647d9eacffe9e85120c4aa7d72c4-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for b00188237ffd9111837bc01eb8689d2e957a1ae0:
  - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1445-1e9c025c5864c1dc0f1fc5d59c1e5de55ca4ef98-arm64_v8a.apk)

Other architectures: [armv7](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1445-1e9c025c5864c1dc0f1fc5d59c1e5de55ca4ef98-armv7.apk), [x86_64](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1445-1e9c025c5864c1dc0f1fc5d59c1e5de55ca4ef98-x86_64.apk), [x86](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1445-1e9c025c5864c1dc0f1fc5d59c1e5de55ca4ef98-x86.apk)

File: .github/workflows/continuous_integration.yml
Comment (suricactus) [inline .github/workflows/continuous_integration.yml:133]:
```suggestion
          ./scripts/basic_upload_apks_service_account.py ${APP_PACKAGE_NAME} alpha "Update from commit ${GITHUB_SHA}" \
              qfield-dev-${{ steps.vars.outputs.CI_UPLOAD_ARTIFACT_ID}}-${{ steps.vars.outputs.CI_COMMIT }}-arm64_v8a.apk \
              qfield-dev-${{ steps.vars.outputs.CI_UPLOAD_ARTIFACT_ID}}-${{ steps.vars.outputs.CI_COMMIT }}-armv7.apk \
              qfield-dev-${{ steps.vars.outputs.CI_UPLOAD_ARTIFACT_ID}}-${{ steps.vars.outputs.CI_COMMIT }}-x86_64.apk \
              qfield-dev-${{ steps.vars.outputs.CI_UPLOAD_ARTIFACT_ID}}-${{ steps.vars.outputs.CI_COMMIT }}-x86.apk
```

File: .github/workflows/continuous_integration.yml
Comment (m-kuhn) [inline .github/workflows/continuous_integration.yml:132]:
```suggestion
          pip install google-api-python-client google-auth-httplib2 google-auth-oauthlib oauth2client
          
```

**Review by suricactus — APPROVED** [review]
Probably wait for the second review, since I am not an expert in deploying apks :)

**Review by 3nids — APPROVED** [review]

**Comment by m-kuhn** [comment]
Great.

Next job: publish tagged releases

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 796d4cadf2aaffce4d57f252b616b35f29aa642c:
  - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1445-13e962241227ecf9856b6517cf3dd4b8afa8fc54-arm64_v8a.apk)

Other architectures: [armv7](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1445-13e962241227ecf9856b6517cf3dd4b8afa8fc54-armv7.apk), [x86_64](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1445-13e962241227ecf9856b6517cf3dd4b8afa8fc54-x86_64.apk), [x86](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1445-13e962241227ecf9856b6517cf3dd4b8afa8fc54-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 3408a69e57a9ce1812bcb6c37aaa533b35033655:
  - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1445-fdc82cd9f6ab0993aca9b26fac374fff73e71ae3-arm64_v8a.apk)

Other architectures: [armv7](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1445-fdc82cd9f6ab0993aca9b26fac374fff73e71ae3-armv7.apk), [x86_64](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1445-fdc82cd9f6ab0993aca9b26fac374fff73e71ae3-x86_64.apk), [x86](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1445-fdc82cd9f6ab0993aca9b26fac374fff73e71ae3-x86.apk)

**Pull request merged**
